developing-packages.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: page
+title: Developing Packages & Plugins
+permalink: /developing-packages/
+---
+
+* TOC
+{:toc}
+
+## Package introduction
+
+Dart packages enable the creation of modular code that can easily be shared. A
+minimal package with a single library consists of:
+
+* A `pubspec.yaml` file: A metadata file that declared the package name,
+  version, author, etc.
+
+* A `lib` directory containing the public code in the package, minimally a
+  single `<package-name>.dart` file.
+
+For additional details, see the [Dart library package](https://www.dartlang.org/guides/libraries/create-library-packages) documentation.
+
+### Package types {#types}
+
+Flutter supports several kinds of packages:
+
+* *Dart packages*: General packages written in Dart without any dependencies on
+  any specific frameworks. They can be used across many frameworks such as
+  [AngularDart](https://webdev.dartlang.org/angular) and Flutter. A concrete
+  example is the [`path`](https://pub.dartlang.org/packages/path) package.
+
+* *Flutter packages*: General packages which contain Flutter specific
```

Comment:
I'm not sure we want to make this distinction. The _kind_ of package is the same: a pub package.

I think the user cares about one thing:

* where do I go to see all the packages I can use as a Flutter developer

The differences you talk about here are implementation details that probably don't matter. Only: "will it work with Flutter?"